src/cmd/manage_nsfs.js

```diff
@@ -99,11 +99,15 @@ async function main(argv = minimist(process.argv.slice(2))) {
         config_root_backend = argv.config_root_backend ? String(argv.config_root_backend) : config.NSFS_NC_CONFIG_DIR_BACKEND;
 
         await check_and_create_config_dirs();
-        const from_file = argv.from_file ? String(argv.from_file) : '';
+        const path_to_json_options = argv.from_file ? String(argv.from_file) : '';
+        let user_input; // in case of add action can be from argv or from-file
         if (type === TYPES.ACCOUNT) {
-            await account_management(argv, from_file);
+            // currently we use from_file only in add action
```

Comment:
hmm, it's still not exactly what I meant.. 
I suggest merging between get_options_from_file() and validate_flags_arguments() in order to get a single flow of user input validation

Reply:
I don't understand why - the number of properties is the same if we would iterate once (outside the from-file and inside the file):
1. First, we validate the command, for example: `sudo node src/cmd/manage_nsfs account add --from_file <path>` that the command has only the flags that we allowed ('from_file', 'config_root_backend', 'config_root') with the right type.
2. Second, we validate what is inside the file "options JSON file" - all the values that we currently have in `account add`.

Reply:
I think that if I will not do it in two steps I will not know the origin of the property (command-line or inside the file).

Reply:
@shirady why do you need to know the origin of the property? 
what I mean is that if the origin of the input is argv or from a file, it doesn't really matter and it would be great to have the validations in a single flow

Reply:
I separated the validation into 2 iterations:
1. On the command: `sudo node src/cmd/manage_nsfs account add --from_file <path>`, here I don't allow all the flags, but only `from_file`, `config_root_backend`, `config_root`). The origin of this check is on argv, if I have all the options extracted - how will I validate this?
2. On the file options - I reuse VALID_OPTIONS_ACCOUNT with the option 'add'.